### PR TITLE
Upload

### PR DESCRIPTION
**Pull request by jeankayy**
integrated add resume button into sign up page

### CODE REVIEW

**Review by andronicus217 — APPROVED** [review]
Looks good!

File: client/src/pages/SignUp.js [hunk 1]
```diff
@@ -109,65 +135,73 @@ const SignUp = () => {
...
+          ? <p> Resume Uploaded </p>
+          : <>
+            <Button variant='contained' onClick={() => fileInputRef.current.click()}>
+              Upload A Resume
+            </Button>
+            <input onChange={handleUpload} multiple={false} ref={fileInputRef} type='file' hidden />
+            </>
+          }
           <button onClick={register} disabled={loading}>Create Account</button>
           <button onClick={register} disabled={loading}>Create Account</button>
```
Comment (sbirvin1s) [inline client/src/pages/SignUp.js]: We should probably get rid of one of these buttons. Currently two are being rendered to the "seeker" view of the form

**Review by sbirvin1s — CHANGES_REQUESTED** [review]
Once the extra button that is being rendered in the seeker view is removed this should be good to go.

> one additional thing I noticed -> If a person is on the form and uses the browser "back" action to leave the page the form view will not reset when they click back into the page. However, the radial button that indicates which account type they have selected (seeker or recruiter) will not be correctly selected. This is minor at the moment, but could potentially lead to some issues in the future

**Review by sbirvin1s — APPROVED** [review]
🔥 Looks good!

**Pull request merged**